src/main/sql/build-db.sql

```diff
@@ -0,0 +1,24 @@
+DROP DATABASE IF EXISTS alert_manager;
+
+CREATE DATABASE alert_manager;
+
+USE alert_manager;
+
+create table subscriptions (
```

Comment:
Small request--use singular `subscription` name to keep it consistent with our other table names.